ext/node/polyfills/internal_binding/cares_wrap.ts

```diff
@@ -107,13 +106,6 @@ export function getaddrinfo(
       });
     }
 
-    // TODO(@bartlomieju): Forces IPv4 as a workaround for Deno not
-    // aligning with Node on implicit binding on Windows
-    // REF: https://github.com/denoland/deno/issues/10762
-    if (isWindows && hostname === "localhost") {
-      addresses = addresses.filter((address) => isIPv4(address));
-    }
```

Comment:
Is this now working correctly?

Reply:
It works now because we implemented `autoSelectFamily` option and that is enabled by default.

Also this workaround is now problematic as `Deno.serve({ hostname: "localhost" })` listens with host `::1`, and `http.connect({ hostname: "localhost" })` can't connect to it with this workaround.